### PR TITLE
testing works, v1 is done

### PR DESCRIPTION
**Pull request by antoinerichez**

### CODE REVIEW

**Comment by antoinerichez** [comment]
make it work, make it right, make it fast

**Review by BertSchoovaerts — APPROVED** [review]
aw yeah 😄

File: src/BikeDataProject.API/Controllers/TrackController.cs [hunk 1]
```diff
@@ -33,10 +33,11 @@ public IActionResult ReceiveGpsTrack([FromBody]Track track, [FromQuery]bool? tes
             try
             {
                 var contribution = locations.ToContribution();
-                this._dbContext.AddContribution(contribution);
                 var userId = this._dbContext.GetUserId(track.UserId);
                 if (userId != 0)
```
Comment (MeurillonGuillaume) [inline src/BikeDataProject.API/Controllers/TrackController.cs:37]: What if the `userId` is 0?

Comment (antoinerichez) [reply src/BikeDataProject.API/Controllers/TrackController.cs:37]: it can't

**Review by MeurillonGuillaume — APPROVED** [review]

**Pull request merged**